### DISCUSSION TITLE
Install termux using the .so files only

### DISCUSSION THREAD

**Discussion by denisidoro** [termux/termux-app#2276] [Q&A] [answered]
Is there a way to "install" termux given its .so files only?

Use case: I'm trying to use termux inside https://github.com/dockcross/dockcross

I tried the following to no avail:

```
❯ ./dockcross-android-arm64 bash -c 'export LD_LIBRARY_PATH="$PWD/lib/arm64-v8a"; ls $LD_LIBRARY_PATH; pkg'
libtermux-bootstrap.so libtermux.so
bash: line 1: pkg: command not found
```

**Comment by agnostic-apollo** [comment, marked as answer]
The `libtermux-bootstrap.so` is technically a zip file extracted by `termux-app` in `$PREFIX`.
The [`libtermux.so`](https://github.com/termux/termux-app/blob/master/terminal-emulator/src/main/jni/termux.c) is a lib used by the `termux-app` for `JNI` interaction for the terminal emulator.

What you are trying to do is not gonna work.
